scripts/restore

```diff
@@ -63,12 +64,12 @@ ynh_system_user_create --username=$app
 # RESTORE THE PHP-FPM CONFIGURATION
 #=================================================
 
-ynh_restore_file --origin_path="/etc/php/7.0/fpm/pool.d/$app.conf"
+ynh_restore_file --origin_path="/etc/php/$phpversion/fpm/pool.d/$app.conf"
 
```

Comment:
```suggestion

ynh_add_fpm_config --phpversion=$YNH_PHP_VERSION
```

Reply:
I see a pattern with me missing to replace php in the restore script twice :)